2.0/alpine/Dockerfile

```diff
@@ -36,6 +36,10 @@ RUN set -eux; \
 	chmod +x /usr/bin/caddy; \
 	caddy version
 
+# set up nsswitch.conf for Go's "netgo" implementation
+# - https://github.com/docker-library/golang/blob/master/1.14/alpine3.12/Dockerfile#L9
```

Comment:
Permalink I guess: https://github.com/docker-library/golang/blob/1eb096131592bcbc90aa3b97471811c798a93573/1.14/alpine3.12/Dockerfile#L9

Reply:
yeah I originally did that but the line was real long 😂 - I'll flip it back.